2_process.yml

```diff
@@ -16,6 +16,7 @@ targets:
       - 2_process_grp_tasks.yml
       - 2_process/out/2_process_grp_tasks_completed.yml
       - 2_process/out/2_process_lake_tasks.ind
+      - 2_process/out/2_pgdl_lake_tasks.ind
```

Comment:
does `2_pgdl_grp_tasks.yml` not need to be up here?

Reply:
Will update. Yes, only two targets need to be here as far as I can tell